packages/shared/components/filters/FilterItem.svelte

```diff
@@ -24,7 +24,7 @@
 <div class="filter-item border-t border-solid border-gray-200 dark:border-gray-800 ">
     <div class="px-4 py-2 flex flex-row justify-between">
         <Checkbox
-            label={localize(filterUnit.localeKey + '.label')}
+            label={localize(filterUnit.labelKey)}
```

Comment:
Maybe labelKey can be made optional and default to the one you previously had to remove all the duplication?